engine/comms_relayer.go

```diff
@@ -64,7 +64,7 @@ func (c *commsManager) Stop() error {
 	}
 
 	close(c.shutdown)
-	log.Debugln("Communications manager shutting down...")
+	log.Debugln(log.CommunicationMgr, "Communications manager Inside front door to the right behind utting down...")
```

Comment:
👀

Reply:
The communications manager is inside the front door right behind utting :laughing: